### ISSUE TITLE
docs: Issue in page ScyllaDB FAQ - out of date questions and answers

### ISSUE THREAD

**Issue by tzach** [scylladb/scylladb#16631]
I would like to report an issue on page https://opensource.docs.scylladb.com/branch-5.4/faq

### Problem
Many of the FAQ entries need to be updated or removed.

>Setting io.conf configuration for HDD storage

This section is out of date, and need refresh

>After upgrade from Scylla 2.1 and older, scylla_reactor_utilization metrics is at high percentage (high CPU utilization is observed). Why does this happen?. 

Old, non-supported releases. It should be removed.

> What if I get an error when trying to use cqlsh DESCRIBE commands?. 

Fixed. It should be removed.

> Check and update Ubuntu 14.04 kernel. 

Old, non supported OS. It should be removed.

>in-memory tables - deprecated Enterprise feature. 

It should be removed.

- I deleted data from Scylla, but disk usage stays the same. Duplication with a similar question in the storage section.
>Do I need to run a tool like upgradesstables when moving to a new format?
> Unlike Apache Cassandra, Scylla does not ship with upgradesstables, a tool that converts SSTables to newer formats. 

[upgradesstables.html](https://opensource.docs.scylladb.com/stable/operating-scylla/nodetool-commands/upgradesstables.html) is part of Scylla.

> The Apache Software Foundation copyright

Irrelevant for this page. It should be removed.

**Comment by tzach** [comment]
Maybe we should kill this page and move relevant questions to the forum.
Its easier to update, and more active.
@annastuchlik @guy9 @timkoopmans what do you think?

**Comment by timkoopmans** [comment]
> Maybe we should kill this page and move relevant questions to the forum. Its easier to update, and more active. @annastuchlik @guy9 @timkoopmans what do you think?

+1 for the forum

**Comment by guy9** [comment]
I agree, the forum is a better location for such content. @annastuchlik let me know once you're ready to move it, we can work on it together.
